pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -42,6 +42,7 @@ import (
 // +kubebuilder:printcolumn:name="Message",type=string,JSONPath=`.status.message`,description="Message"
 // +kubebuilder:printcolumn:name="Health",type=string,JSONPath=`.status.ceph.health`,description="Ceph Health"
 // +kubebuilder:printcolumn:name="External",type=boolean,JSONPath=`.spec.external.enable`
+// +kubebuilder:printcolumn:name="FSID",type=string,JSONPath=`.status.ceph.fsid`,description="Ceph FSID"
```

Comment:
I don't believe you can have a space in the name.
```suggestion
// +kubebuilder:printcolumn:name="FSID",type=string,JSONPath=`.status.ceph.fsid`,description="FSID"
```

This is what I see what I tested your change:
```
% kn get cephcluster
NAME         DATADIRHOSTPATH   MONCOUNT   AGE   PHASE   MESSAGE                        HEALTH      EXTERNAL   FSID
my-cluster   /var/lib/rook     3          42h   Ready   Cluster created successfully   HEALTH_OK              9341d8c4-06e8-4c24-9451-dffd04600cac
```

Reply:
`"Ceph FSID"` is description, not name

Reply:
Ha! I overlooked that name. LGTM